src/table.ts

```diff
@@ -1507,13 +1508,26 @@ Please use the format 'prezzy' or '${instance.name}/tables/prezzy'.`);
     const onBatchResponse = (
       err: ServiceError | PartialFailureError | null
     ) => {
-      // TODO: enable retries when the entire RPC fails
       if (err) {
-        // The error happened before a request was even made, don't retry.
+        // Retry RPC level errors
+        if (!(err instanceof PartialFailureError)) {
+          const serviceError = err as ServiceError;
```

Comment:
should this be `if (err instanceof ServiceError)`?

Reply:
I couldn't do `if (err instanceof ServiceError)` because ServiceError is an interface so I can't do instanceof on it 🤔